### PR TITLE
LTD-719: Email reports using Django SMTP Email backend

### PR DESCRIPTION
**Pull request by saruniitr**
## Change Description

Add changes to create a background task which runs the query, generates report and saves it in a temporary file and then emails it to the Users using SMTP Emailbackend.

Sending emails is behind a feature flag as it is temporary which is by default turned off.
Recipient list is configurable using env variables.

### CODE REVIEW

File: api/reports/tasks.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+                        "report_filename": temp_file.name.split("/")[-1],
+                        "link_to_file": prepare_upload(f, is_csv=True),
+                    }
+                    response = notify_client.send_email(recipient, settings.LITE_REPORTS_EMAIL_TEMPLATE_ID, data)
+                    logger.info(response)
+        except errors.HTTPError as err:
+            raise err
+
+
+@background(queue=EMAIL_REPORTS_QUEUE, schedule=0)
```
Comment (r4vi) [inline api/reports/tasks.py:103]: won't this be constantly running instead of 7am every day?

Comment (saruniitr) [reply api/reports/tasks.py:103]: Forgot to change this, it is like this only for testing

File: api/reports/views.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from django.http import HttpResponse
+from rest_framework import status
+from rest_framework.views import APIView
+
+
+from api.reports.tasks import email_reports_task
+
+
+class EmailReportView(APIView):
```
Comment (r4vi) [inline api/reports/views.py:9]: do we want to disable/remove the view?

**Review by r4vi — APPROVED** [review]

File: api/reports/views.py (hunk 2 above)
Comment (saruniitr) [reply api/reports/views.py:9]: ah, yes will disable the route in the urls.py

**Pull request merged**